src/Tabs/TabList.tsx

```diff
@@ -0,0 +1,21 @@
+/* @jsx jsx */
+
+import { css, jsx } from '@emotion/core';
+import * as React from 'react';
+import { TabList } from 'react-tabs';
+
+const StyledTabList: React.FunctionComponent & { tabsRole: string } = props => (
```

Comment:
`react-tabs` seems to export props, so let's make use of them.
```suggestion
const StyledTabList: React.FunctionComponent<TabListProps> & { tabsRole: string } = props => (
```